cmd/atc-installer/installer/run.go

```diff
@@ -23,17 +23,18 @@ import (
 )
 
 type Config struct {
-	Labels                 map[string]string `json:"labels,omitempty"`
-	Annotations            map[string]string `json:"annotations,omitempty"`
-	Image                  string            `json:"image,omitzero" Description:"set the image you want to deploy"`
-	Version                string            `json:"version,omitzero" Description:"version of the deployed image"`
-	Port                   int               `json:"port,omitzero"`
-	ServiceAccountName     string            `json:"serviceAccountName,omitzero"`
-	ImagePullPolicy        corev1.PullPolicy `json:"imagePullPolicy,omitzero"`
-	GenerateTLS            bool              `json:"generateTLS,omitzero" Description:"generate new tls certificates even if they already exist"`
-	DockerConfigSecretName string            `json:"dockerConfigSecretName,omitzero" Description:"name of dockerconfig secret to allow atc to pull images from private registries"`
-	LogFormat              string            `json:"logFormat,omitzero" Enum:"json,text"`
-	Verbose                bool              `json:"verbose,omitzero" Description:"verbose logging"`
+	Labels                    map[string]string `json:"labels,omitempty"`
+	Annotations               map[string]string `json:"annotations,omitempty"`
+	Image                     string            `json:"image,omitzero" Description:"set the image you want to deploy"`
+	Version                   string            `json:"version,omitzero" Description:"version of the deployed image"`
+	Port                      int               `json:"port,omitzero"`
+	ServiceAccountName        string            `json:"serviceAccountName,omitzero"`
+	ImagePullPolicy           corev1.PullPolicy `json:"imagePullPolicy,omitzero"`
+	GenerateTLS               bool              `json:"generateTLS,omitzero" Description:"generate new tls certificates even if they already exist"`
+	DockerConfigSecretName    string            `json:"dockerConfigSecretName,omitzero" Description:"name of dockerconfig secret to allow atc to pull images from private registries"`
+	LogFormat                 string            `json:"logFormat,omitzero" Enum:"json,text"`
+	Verbose                   bool              `json:"verbose,omitzero" Description:"verbose logging"`
+	ValidationWebhookTimeout  *int32            `json:"validationWebhookTimeout,omitzero" Description:"timeout in seconds for validation webhooks (default: 10)"`
```

Comment:
I think a pointer is not necessary in this case. If the value is 0 or less than 0 we can ignore the field.